temporalio/bridge/src/worker.rs

```diff
@@ -20,6 +23,12 @@ pub struct WorkerRef {
     worker: Option<Arc<temporal_sdk_core::Worker>>,
 }
 
+#[pyclass]
+pub struct ReplayWorker {
```

Comment:
Ah, yeah I forgot to delete it. The tuple is actually necessary, because I can't move out fields from the class, they have to be clone-able for a getter to work which makes everything all hard.

Reply:
Not following the cloneable thing exactly, but if tuple required, no prob.